cypress/integration/e2e/commercial.spec.js

```diff
@@ -36,12 +37,13 @@ describe('Commercial E2E tests', function () {
             runLongReadTestFor(`Article?url=${longReadURL}`);
         });
 
-        // Uncomment to check Locally.
-        // It seems overriding the baseURL works fine locally but hangs on CI.
-
-        // it(`It should check slots for a long article in Frontend`, function () {
-        //     Cypress.config('baseUrl', '');
-        //     runLongReadTestFor(`${longReadURL}?dcr=false`);
-        // });
+        // Skipping only on CI because, overriding the baseURL works fine locally but hangs on CI.
+        // eslint-disable-next-line mocha/no-setup-in-describe
+        skipOn(Cypress.env('CI') === 'true', () => {
```

Comment:
thanks @mxdvl , @oliverlloyd , seems like it didn't skip it! 🤔 
Is 'CI' the right name here?

Reply:
Sorry, you need to add the `env` via the plugins!

https://github.com/guardian/consent-management-platform/blob/95d97ecb13c45158bf9b1fae0cc284eb588653e1/cypress/plugins/index.js

Reply:
Thanks ! Thank, and replacing `CI` to `TEAMCITY` has fixed it